test/e2e/node_reuse_test.go

```diff
@@ -42,7 +42,7 @@ func nodeReuse() {
 	)
 
 	const (
-		artifactoryURL = "https://artifactory.nordix.org/artifactory/airship/images/k8s"
+		artifactoryURL = "https://artifactory.nordix.org/artifactory/metal3/images/k8s"
```

Comment:
Please update `artifactoryURL` in the `release-0.5` branch as well (we do not have e2e tests in the release-0.4 branch) only.